app/controllers/tasks.js

```diff
@@ -36,7 +36,7 @@ export default class TasksController extends Controller {
   }
 
   constructReqBody(object) {
-    const requestBody = { status: object.status };
+    const requestBody = { ...object };
```

Comment:
please provide details in the PR description about what was happening because of sending it like this, and how is your solution solving it